packages/utilities/package.json

```diff
@@ -48,5 +48,17 @@
   "peerDependencies": {
     "@types/react": ">=16.8.0 <18.0.0",
     "react": ">=16.8.0 <18.0.0"
+  },
+  "exports": {
+    ".": {
+      "types": "./lib/index.d.ts",
+      "import": "./lib/index.js",
+      "require": "./lib-commonjs/index.js"
+    },
+    "./lib/sessionStorage": {
```

Comment:
This is an example of a weird export; sessionStorage probably should be moved out, or exported from index, but right now it's not exported so this entry is required without further modifications to the package.